spell out "MR" -> Merge Request

bquast opened the pull request:
The only abbreviation in this file is "MR".

It would be cleaner to spell this out.

callmetango commented:
Yes, incorrect wording as explained in [Pull Request vs. Merge Request | Simplilearn](https://www.simplilearn.com/pull-vs-merge-request-definition-differences-benefits-article).

And incorrect speling too: It should be `pull request` since pull request is not a name in this context.

callmetango commented:
@bquast You missed the "MRs" in

> 3. Once release is out, bugfixes should be submitted separately, against
   the affected release branch(es) as well as master (so multiple MRs)

:wink:

bquast commented:
> Actually, the "Merge Request" is incorrect term in the context of GitHub. "Merge Request"s are GitLab's entities - they're called "Pull Request"s on GitHub

Updated.

> thanks. editorial note: please add "CONTRIBUTION.md:" prefix to commit message.

Thank you, done.  BTW, this could be item 6. in CONTRIBUTION.md (prefix commit msg with filename)

> @bquast You missed the "MRs" in
> 
> > 3. Once release is out, bugfixes should be submitted separately, against
> >    the affected release branch(es) as well as master (so multiple MRs)
> 
> 😉

Thanks for pointing this out. Updated.

> Yes, incorrect wording as explained in [Pull Request vs. Merge Request | Simplilearn](https://www.simplilearn.com/pull-vs-merge-request-definition-differences-benefits-article).
> 
> And incorrect speling too: It should be `pull request` since pull request is not a name in this context.

Thanks, I chagned both to lowercase.

metux commented:
maybe squash them all together instead of having series of commits changing the same words over and over ?

bquast commented:
> maybe squash them all together instead of having series of commits changing the same words over and over ?

Agreed, it was a bit much. Done.
